src/algebra/group_ring_action.lean

```diff
@@ -40,7 +40,7 @@ class mul_semiring_action (M : Type u) (R : Type v) [monoid M] [semiring R]
 
 section semiring
 
-variables (M G : Type u) [monoid M] [group G]
+variables (M G K N : Type u) [monoid M] [group G] [semiring K] [semiring N]
```

Comment:
You only need `monoid` not `semiring` here. You should be able to use the existing `M` as one of the monoids.

Reply:
OK, thank you. I just changed those